show.php

```diff
@@ -147,7 +147,7 @@ function boxeePlay($filename) {
     }
 
     // limit random to not unseen movies only
-    if ($config['showrandomunseen'])
+    if (array_key_exists( 'showrandomunseen', $config ))
```

Comment:
Same here, to keep functionality something like:
`if (array_key_exists( 'showrandomunseen', $config) && $config['showrandomunseen'])`

Reply:
code amended